src/libsyntax/tokenstream.rs

```diff
@@ -494,16 +459,13 @@ impl Cursor {
             return;
         }
         let index = self.index;
-        let stream = mem::replace(&mut self.stream, TokenStream(None));
+        let stream = mem::replace(&mut self.stream, TokenStream::empty());
```

Comment:
```suggestion
        let stream = mem::take(&mut self.stream);
```
Assuming `Default` is implemented.